README.md

```diff
@@ -1,306 +1,184 @@
-## Go Ethereum
+Prysmatic Labs Sharding Implementation
+==========================
+This is the main repository for the sharding implementation of the go-ethereum client by [Prysmatic Labs](https://prysmaticlabs.com). For the original, go-ethereum project, refer to the following [link](https://github.com/ethereum/go-ethereum).
```

Comment:
Could you run some markdown linter to take care of these really long lines? I think a line length of 100 is appropriate and helps in readability of this file.